fix of helm installation command in doc (#3333)
googleforgames/agones: site/content/en/docs/Installation/Install Agones/helm.md

apiVersion: cert-manager.io/v1
kind: Certificate
metadata:
  name: my-release-agones-cert
  namespace: agones-system
spec:
  dnsNames:
    - agones-controller-service.agones-system.svc
  secretName: my-release-agones-cert
  issuerRef:
    name: selfsigned
    kind: ClusterIssuer
EOF
```

> After the certificates are generated, we will want to [inject caBundle](https://cert-manager.io/docs/concepts/ca-injector/) into controller and extensions webhook and disable controller and extensions secret creation by using yaml file. This process avoids the cluter of passing too many parameters to helm install command. Please refer below `file_name.yaml` file:

Reviewer: Nit: some sentence structure.

```suggestion
After the certificates are generated, we will want to [inject caBundle](https://cert-manager.io/docs/concepts/ca-injector/) into the controller and extensions webhook and disable the controller and extensions secret creation through the following values.yaml file.:
```

> ```

Reviewer: Just for some syntax highlighting.

```suggestion
```yaml
```

    - key: ca.crt
      path: ca.crt
    - key: tls.crt
      path: server.crt
    - key: tls.key
      path: server.key
    allocationApiService:
      annotations:
        cert-manager.io/inject-ca-from: agones-system/my-release-agones-cert
      disableCaBundle: true
    validatingWebhook:
      annotations:
        cert-manager.io/inject-ca-from: agones-system/my-release-agones-cert
      disableCaBundle: true
    mutatingWebhook:
      annotations:
        cert-manager.io/inject-ca-from: agones-system/my-release-agones-cert
      disableCaBundle: true
  extensions:
    disableSecret: true
    customCertSecretPath:
    - key: ca.crt
      path: ca.crt
    - key: tls.crt
      path: server.crt
    - key: tls.key
      path: server.key
    allocationApiService:
      annotations:
        cert-manager.io/inject-ca-from: agones-system/my-release-agones-cert
      disableCaBundle: true
    validatingWebhook:
      annotations:
        cert-manager.io/inject-ca-from: agones-system/my-release-agones-cert
      disableCaBundle: true
    mutatingWebhook:
      annotations:
        cert-manager.io/inject-ca-from: agones-system/my-release-agones-cert
      disableCaBundle: true
```

> After, creating yaml file use below command to install Agones:

Reviewer: ```suggestion
After copying the above yaml into a `values.yaml` file, use below command to install Agones:
```

```bash

> helm install my-release --namespace agones-system --create-namespace --values /path/file_name.yaml agones/agones

Reviewer: ```suggestion
helm install my-release --namespace agones-system --create-namespace --values values.yaml agones/agones
```

Author: Thanks for suggestions! I have updated the PR.

```bash
helm install my-release \
  --set agones.controller.disableSecret=true \

>   --set-string "agones.controller.customCertSecretPath[0].key=ca.crt" \

Reviewer: I'm actually thinking - let's remove this top section with all the `--set` commands, I don't think anyone would do it in practice, and it's far easier to read with the yaml file.

So let's just keep the YAML file version (unless anyone disagrees?)

Author: Sure, changes added!
